xla/service/gpu/gemm_rewriter_triton_test.cc

```diff
@@ -57,7 +57,8 @@ ENTRY e {
     lhs_contracting_dims={1}, rhs_contracting_dims={0}
 })")
                     .value();
-  EXPECT_TRUE(GemmRewriterTriton({se::CudaComputeCapability::AMPERE, 0})
+  GpuVersion gpu_version;
```

Comment:
It looks to me that this is a non-functional change. At least I don't see that GpuVersion is automatically initialized to AMPERE, 0.